lexicon/tests/providers/test_ultradns.py

```diff
@@ -0,0 +1,19 @@
+"""Integration tests for UltraDNS"""
+from unittest import TestCase
+
+from lexicon.tests.providers.integration_tests import IntegrationTests
+
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class UltradnsProviderTests(TestCase, IntegrationTests):
+    """TestCase for UltraDNS"""
+    provider_name = 'ultradns'
+    domain = 'example-abtest.com'
+
+    def _filter_headers(self):
+        return ['Authorization']
+
+    def _filter_post_data_parameters(self):
+        return ['username', 'password']
```

Comment:
It seems that parameter `accessToken` in the returned JSON during an authentication would also be useful to be filtered, if someone refresh the data, but using a user+password credentials this time.

Reply:
Fixed